src/test/shell/integration/aquery_test.sh

```diff
@@ -1479,6 +1479,7 @@ EOF
   cat output >> "$TEST_log"
   assert_not_contains "actions" output
 
+  bazel build --nobuild "//$pkg:foo"
```

Comment:
out of curiosity -- why are these needed?

Reply:
Skyframe state will be dirty after the first build since that build modified the lockfile. That's why commands that directly dump Skyframe state will not give interesting results until after a command that didn't change the lockfile. This is similar to the types of issues we had when we introduced the regular lockfile.

Reply:
I was also wondering this in the internal review, will add this as a comment for those places.